frontend/src/data/Dialect.ts

```diff
@@ -32,9 +27,9 @@ export default class Dialect {
     this.firstPublicationDate = firstPublicationDate;
   }
 
-  async fetchReport(baseURI: URI = siteURI) {
-    const url = baseURI.clone().filename(this.path).suffix("json").href();
-    const response = await fetch(url);
+  async fetchReport(baseURL: URL = siteURL) {
+    const url = new URL(this.path + ".json", baseURL.href);
```

Comment:
I believe this behaves differently in some cases:
- old way: `https://harrel56.github.io/bowtie` resolves to  `https://harrel56.github.io/bowtie/draftX`
- new way:  `https://harrel56.github.io/bowtie` resolves to  `https://harrel56.github.io/draftX` - which is just wrong

Possibly there are differences in other parts too. Certainly, before merging it should be checked if GH pages on forks works correctly